source/_posts/2016-10-12-third trial.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "Third trial"
+date: "2016-10-12 07:11:21"
+categories: tech-stories
+authors: ["Rose"]
+---
+At [ePages](http://www.epages.com/us/) we provide and constantly improve our existing cloud-based online shop software and therefore count with outstanding software developers and of course a CTO who is dedicated to xxx.
+
+**Rose: What key skills are necessary for a developer to successfully contribute to the ePages shop software?**
```

Comment:
@roseritter think I read that before. Isnt that the same as the first one? Could you provide other content?